students/10th/taesung/posting/views.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from user.models  import User
+from .models      import (
+    Article,
+    Comment
+)
+
+from user.models  import User
+
+class ArticleView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            if User.objects.filter(email = data['email']).exists() and not Article.objects.filter(head = data['head']).exists():
+                Article(
+                    email = User.objects.get(name= data['email']),
+                    head = data['head'],
+                    body = data['body']
+                ).save()
+                return JsonResponse({'message':'SUCCESS'}, status=200)
+            else:
+                return JsonResponse({'message': 'INVALID_INFORMATION'}, status=401)
+        except KeyError:
+            return JsonResponse({'messgae' : 'FAIL'}, status=400)
```

Comment:
단순한 Fail 보다는 KEY_ERROR 라고 명시해주는게 좋을 것 같습니다.
```suggestion
            return JsonResponse({'messgae' : 'FAIL'}, status=400)
```